### PR TITLE
Adding final bug-releases for the gromacs-2016 and -2018 and Plumed

### PR DESCRIPTION
**Pull request by hseara**
It makes sense to provide the latest bug releases for the old gromacs-2016 and gromacs-2018 branches. Those branches are still commonly used, perhaps, unfortunately. There are important bugs that have been fixed since the older versions currently provided by spack were released.

### CODE REVIEW

**Comment by hseara** [comment]
I should have done two branches, and make two separate pull requests, but I didn't. Please let me know if you can proceed with them together or do you really need to split them.

File: var/spack/repos/builtin/packages/plumed/package.py [hunk 1]
```diff
@@ -21,9 +21,16 @@ class Plumed(AutotoolsPackage):
     and C/C++ codes.
     """
     homepage = 'http://www.plumed.org/'
-    url = 'https://github.com/plumed/plumed2/archive/v2.5.0.tar.gz'
-
+    url = 'https://github.com/plumed/plumed2/archive/v2.5.3.tar.gz'
+    git = 'https://github.com/plumed/plumed2.git'
+
+    version('develop', branch='master')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/plumed/package.py]:
```suggestion
    version('master', branch='master')
```
See #1983

Comment (hseara) [reply var/spack/repos/builtin/packages/plumed/package.py]: The change seems reasonable.

Comment (hseara) [reply var/spack/repos/builtin/packages/plumed/package.py]: The change you suggest is reasonable. Do you need any action from my side to finalize the commit?

**Pull request merged**